cxplat/cxplat_test/cxplat_fault_injection_test.cpp

```diff
@@ -64,6 +64,7 @@ _is_fault_injection_enabled_in_environment()
     }
 }
 
+#ifndef NDEBUG
```

Comment:
Every time I try to read this, it takes me a minute to parse "if not not debug" to mean "if release". Not sure about other folks, but I think we should have a "release" define that is !NDEBUG to make this easier to parse.

Reply:
Will add CXPLAT_DEBUGGING_FEATURES_ENABLED define to help readability